### DISCUSSION TITLE
Can inkstitch pass the mom test?

### DISCUSSION THREAD

**Discussion by neural-loop** [inkstitch/inkstitch#1067] [General]
Hi all,

My mother has a passion for embroidery and has been seeking information as to the software and hardware requirements to configure and begin using a 9 needle (I think) machine that she has acquired. She mentioned the software packages needed to run the machines cost about $1500, and so I asked her that we investigate inkstitch as a possibility. 

At first she was very skeptical, saying that when she worked in the shop that people that digitized with other software often came in with many issues and that she had concerns about the quality. I described to her a bit that this is a bit different, with open software and efforts from many passionate stitchers. After seeing some of the produced digitization samples, she agreed that they were special and that was when she said she would give it a look. 

The next important question, was whether she could open the design files (She works from a collection of DST files that she modifies) to send to her machine. We looked over the features and saw that DST reading and writing is supported. 

Finally, we looked over the blog of @Augusa (though we didn't translate) and she saw that people were sharing notes and tips on the digitization process, she started to become excited that she can begin working with her machine sooner than she thought, possibly not needing to gather up the $$ for licenses.

I've asked her to watch some of the tutorial videos and to try to develop some understanding, and that if this is able to work with her machine, it could be very nice. 

- My question, after working with the commercial software, is this going to be a difficult transition? 
- Are there things for professional use that would be a show stopper for someone? 
- Is there a community that can fill gaps in some of the technical areas and that she might interact with to share knowledge (I see there is here, but github may be a bit intimidating initially) 
- Any other words of advice or suggestions that I can pass on to her?
- Is it an issue that the embroidering machine is probably 23+ years old (the manual shows it uses some kind of PCI card with serial interface - I don't know exactly the details yet) 

Mostly, I just want her to have a good experience and hope that she can find the benefits of open source community vs the large commercial packages and appreciate it. In the moment I'm in a different country and can't provide the hands on support, but will do what I can to help out in the technology area. From everything I've read on the website, it seems that this could be a perfect place where her interests and my appreciation of open source overlap and I'm hopeful to invite her to this community :) 

Thanks!

**Comment by wwderw** [comment]
I would be willing to help out as much as I can.  

I have been digitizing for embroidery since summer of 94 and have used everything from Wilcom, DRAWings, Wings, PE-Designs, PFAFF 5D.

On YouTube I did a short video on if Ink/Stitch was a viable replacement from something like EmbroideryStudio, what to consider etc.

There is a curve, like with everything else, but if I can help, I'll do my best to ease the transition.

**Comment by LochanLoch** [comment]
I am not good in English or French writing or reading ... but on Facebook groups/Inkstitchfrance is very active and helpful.

**Comment by lexelby** [comment]
I think the answer, as with many things is: maybe. :)

I would love for your mother, and anyone else, to be able to use Ink/Stitch for all of their digitization.  I do, and I've never used anything else.  I wrote it specifically because of that $1500+ price tag.  But Ink/Stitch has a lot of rough edges and it's under active development.  There are also a lot of features we want to have that we haven't added yet (radial fill, fill stitch patterns, etc).

I think she should go for it.  What's the harm?  Even if it doesn't work out, it won't be a waste of money. ;)  But more importantly, if it doesn't work out, we want to hear why, since we're constantly looking to improve.

As to her machine, I can attest that Ink/Stitch will work just fine.  I also have a 9-needle 23+-year-old machine (Brother BES-916AC) and Ink/Stitch produces DST files that work perfectly with it.

There are at least two active Ink/Stitch Facebook groups.  The French group "Inkscape -- inkstitch" has a few thousand members (!!!).  There's also an English language group that has a smaller user-base but you can quickly get answers to questions.

**Reply by goyia** [reply]
Hello my name is Goyia (mom) from Justin's questions. I was laid off last March, due to covid. My job was then eliminated. I have gathered enough money to purchase 2 Brother machine.
The machines back plate reads BE0901E-AC-SA, Date 2010/03. They will run off a computer. Can anyone give me any specifics about what computer to use, with this information?
I would like for the machines be on a computer separate from the digitizing software. (I believe)
I want to try the ink/stitch digitizing software as well. 
At my former job,I used melco design shop software for the last 14 years. Vector level for the last 5 of those years. When I went to the vector level my computer wasn't able to keep up with my files. When I updated the digitizing file, it would sometimes take several minutes. That was very frustrating and much time wasted waiting for the file to reload. I want to avoid this issue. 
My question is, what laptop computer would be the best for digitizing software? I'm not looking for top of the line, but a good computer with a good size screen. With good CPU and plenty of RAM. I understand that I should have 16-32GB. 
I have no computer knowledge and any suggestions will be appreciated. 
Thanks,
Goyia pronounced Joya 😊

**Reply by lexelby** [reply]
Hi!  Nice to meet you :)

That's the machine I have, or very similar.  Mine says BES-916AC on the nameplate but BE-0901E-AC on the screen when I start it up.

Your machines have floppy drives, right?  Buy a USB floppy drive for your computer.  Then you can save files from Ink/Stitch in DST format into the floppy disk.  Your machines can read that format directly.

As to a computer, most modern computers should be able to run Ink/Stitch fine.  I run it on my laptop with half that amount of RAM.  Higher CPU frequency (GHz) is better for Ink/Stitch, but anything over 2.2GHz should do it.  I always recommend getting a computer with an SSD rather than the old kind of spinning hard drive.

I'd be happy to check out a laptop and give you an opinion if you'd like to post a link here.

**Reply by goyia** [reply]
Thank you for the feedback. The machines run directly from the computer. There's a box of connectors and stuff that comes with the machines. I don't have them presently, but will talk to the guy that I'm buying them from. He used a laptop to run them

**Reply by lexelby** [reply]
All, I see.  I know some of these machines are like that, and some have floppy drives.  I'm afraid I don't know anything about running them from the computer.  Sorry!

**Comment by neural-loop** [comment]
My mom wanted me to write and thank you all for the responses. She is looking forward to using it now, and the notes left here have her on board. I'll have her give an update once she gets going! Thanks
